### DISCUSSION TITLE
Tables within blocks are broken - Pimcore v10.1.3

### DISCUSSION THREAD

**Discussion by awroblewski** [pimcore/pimcore#10774] [Q&A / Support] [answered]
Hello,

creating a field of type table within a block seems to be broken. I can save the class definition, however, an error is raised when I try to actually create the block in a dataobject. No fields are displayed just the headline.

In console the following error is displayed:
_Uncaught TypeError: Cannot read properties of undefined (reading 'length') at klass.initialize (script-proxy_

Creating a table field outside the block works fine.

Did I missed soemthing or is'it a bug?

Pimcore v10.1.3

many thanks

**Comment by dvesh3** [comment, marked as answer]
Already fixed on 10.1.4 https://github.com/pimcore/pimcore/pull/10400

**Reply by awroblewski** [reply]
Hi Divesh,

thanks for the prompt reply. Just updated -> works now.

:)
